packages/blockchain-extension/extension/util/InternalUtil.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+'use strict';
+
+import { ExtensionUtil } from './ExtensionUtil';
+
+/*
+ * This file will hold generic util functions used by the extension.
+ */
+export class InternalUtil {
```

Comment:
not sure i like the name but not sure what to call it

Reply:
Changing to ModuleUtil